packages/xod-client/src/utils/menu.js

```diff
@@ -22,7 +22,7 @@ const rawItems = {
   },
   saveProject: {
     key: 'saveProject',
-    label: 'Save Project',
+    label: 'Save All',
     command: COMMAND.SAVE_PROJECT,
```

Comment:
Should we also rename the command: `SAVE_PROJECT` → `SAVE_ALL`? So that no one could be confused.